environments/development/prison-safety-dsai/sdt-sh-location/workflow.yml

```diff
@@ -1,8 +1,8 @@
 dag:
   repository: moj-analytical-services/airflow-sh-location
-  tag: v4.4.3
+  tag: v4.4.5
   env_vars:
-    AIRFLOW_ENVIRONMENT: "DEVELOPMENT"
+    AIRFLOW_ENVIRONMENT: "PRODUCTION"
```

Comment:
How is this being consumed in the workflow? Just mindful that this is a DAG in `development` in terms of our airflow, so curious what's motivating the change